libr/util/w32-sys.c

```diff
@@ -151,4 +151,20 @@ char *ReadFromPipe(HANDLE fh) {
 	str[strl] = 0;
 	return str;
 }
+
+R_API void print_lasterr (const char *caller, char *cause) {
```

Comment:
all apis in r_util should start with r_ and... what you are doing here is basically the same thing its done by perror, see r_sys_perror for details